multigeiger/webconf.cpp

```diff
@@ -162,47 +159,39 @@ void configSaved(void) {
   tick_enable(true);
 }
 
-void initConfigVariables(void) {
-  format_bool(&speakerTick, speakerTick_c);
-  format_bool(&playSound, playSound_c);
-  format_bool(&ledTick, ledTick_c);
-  format_bool(&showDisplay, showDisplay_c);
-  format_bool(&sendToCommunity, sendToCommunity_c);
-  format_bool(&sendToMadavi, sendToMadavi_c);
-  format_bool(&sendToBle, sendToBle_c);
-  format_bool(&sendToLora, sendToLora_c);
-}
-
 void setup_webconf(bool loraHardware) {
   isLoraBoard = loraHardware;
   iotWebConf.setConfigSavedCallback(&configSaved);
-  iotWebConf.setupUpdateServer(&httpUpdater);
+  iotWebConf.setupUpdateServer(
+  [](const char *updatePath) {
+    httpUpdater.setup(&server, updatePath);
+  },
+  [](const char *userName, char *password) {
+    httpUpdater.updateCredentials(userName, password);
+  });
```

Comment:
this looks a bit strangely formatted.

also, what does `[](...)` even mean? :-)

Reply:
The strange formatting comes from 'astyle'. The code comes from the example or from the migration guide.

Reply:
so astyle would complain if you indented the lines?

Reply:
ah, i see you have solved this already, thanks.